node.js

```diff
@@ -87,13 +84,8 @@ export default class Node extends Tool {
      * Download Node version data. If requests fail, use a local file
      * @returns {Promise<void>}
      */
-    async getVersionData() {
-        try {
-            return await getVersionData()
-        } catch (e) {
-            this.warning(`Failed to download Node version data: ${e.message}`)
-            return nodeVersions
-        }
+    getVersionData() {
+        return nodeVersions
```

Comment:
Add a huge TODO here to add back the logic to fetch node versions from Nodejs

Reply:
updated